pyanaconda/modules/localization/localization.py

```diff
@@ -177,6 +182,25 @@ def get_locale_data(self, locale_id):
 
         return tdata
 
+    def get_locale_keyboard_layouts(self, lang):
+        """Get localized keyboard layouts for a given locale.
+
+        :param lang: locale string (e.g., "cs_CZ.UTF-8")
+        :return: list of dictionaries with keyboard layout information
+        """
+        language_id = lang.split("_")[0].lower()
+        english_name = get_english_name(language_id)
+
+        return [
+            {
+                "layoutId": name,
+                "description": info.desc,
+                "langs": info.langs,
+            }
+            for name, info in self._layout_infos.items()
+            if english_name in info.langs
```

Comment:
This code could be harder to understand especially about the last `if` on the separate line. It a bit looks like:
```
for ...
   if ...
````
with broken intend.

However, it is not strictly bad so I don't insist on changing the code. Just pointing this out.